config/formats.js

```diff
@@ -798,7 +798,7 @@ let Formats = [
 		],
 
 		mod: 'roulettemons',
-		ruleset: ['Standard Pet Mod'],
+		ruleset: ['Standard NatDex', 'Dynamax Clause', 'Sleep Clause Mod', 'Species Clause', 'Moody Clause', 'Evasion Moves Clause', 'Swagger Clause', 'Baton Pass Clause', 'OHKO Clause'],
```

Comment:
I'm surprised Standard NatDex doesn't have these clauses.

Reply:
Wait was there a change or I just hit resolve, my bad

Reply:
National Dex formats have these clauses added separately instead of that for some reason
https://github.com/smogon/pokemon-showdown/blob/master/config/formats.js#L422
```
ruleset: ['Standard NatDex', '+Unobtainable', '+Past', 'OHKO Clause', 'Evasion Moves Clause', 'Species Clause', 'Dynamax Clause', 'Sleep Clause Mod'],
```